srml/staking/src/lib.rs

```diff
@@ -551,9 +551,12 @@ decl_module! {
 		///
 		/// The dispatch origin for this call must be _Signed_ by the controller, not the stash.
```

Comment:
```suggestion
		/// The dispatch origin for this call must be _Signed_ by the stash, not the controller.
```

Reply:
Either I have not grasped the new stash <-> controller mapping or a bunch of other ones also could use an update:

https://github.com/paritytech/substrate/blob/e0bf33e4dd95c2560be397623e295b7073641d82/srml/staking/src/lib.rs#L678

Mentioning that `targets` must be stash accounts, as opposed to the previous controller accounts.
https://github.com/paritytech/substrate/blob/e0bf33e4dd95c2560be397623e295b7073641d82/srml/staking/src/lib.rs#L634